packages/block-library/src/latest-posts/edit.js

```diff
@@ -116,6 +116,8 @@ class LatestPostsEdit extends Component {
 			},
 		];
 
+		const dateFormat = getSettings().formats.date;
```

Comment:
A bit outside the scope of this pull request, but: Is `getSettings` a function we expect to want to continue to support into the future, or would it be preferred that a REST endpoint (e.g. settings endpoint) return this sort of information? We have a _bit_ of it already with `gmt_offset`, `timezone_string` on the root `/wp-json/`, and otherwise have the beginnings of tools to surface this to components (`data`, `core-data`).

I wonder then if, as we look to API finalization, we might want to at the very least [mark it as experimental](https://github.com/WordPress/gutenberg/blob/master/docs/reference/coding-guidelines.md#experimental-apis), and/or mimic the availability of these settings as a set of store selectors (either in `date` or `core-data).

Reply:
Might make sense to mark as experimental. It's straightforward and gives us room to change. @Soean, anything you need from any of us at this point?

Reply:
@mcsf So what ware the to-dos in this PR to get merged?

Reply:
I'd say we can tackle the _experimentalization_ of the API for a second PR. Ideally we'd have it ready for 4.1 (4.2 at the latest).

Is it something you'd be up for, @Soean?

That aside, this looks good, I think I'll 👍 it!